beluga_amcl/README.md

```diff
@@ -0,0 +1,63 @@
+# Beluga AMCL
+
+Beluga AMCL is a ROS 2 node based on [Beluga](../beluga) featuring interface parity with
```

Comment:
In order to better explain what we mean by _ feature parity_ , and the extent to which we support it, maybe adding one of these would be informative:
- A feature table, indicating:
  - name of an AMCL feature
  - whether it's supported by beluga
  - additional notes if needed (e.g.  "beluga does not suspend sensor processing when the robot is stopped").
- A list of deviations from the behavior of AMCL in our implementation.

The first one is probably more impact (_ooohh, it IS a drop in replacement!_), the second one is much shorter.

Reply:
The first option also has the advantage that if someone is looking for a particular feature they are interested in, and we haven't realized it's even there, they can point that that feature is missing.

In the second case it's not clear _which_ features we considered for feature parity, since only the ones that we know are missing or whose behavior is different are listed.

Reply:
Do you mean interface parity as stated in the docs, or do you want to describe feature parity in a separate section?

I like the first option, but identifying AMCL features is not that easy in my opinion, do you have a reference in mind that already describes them? Whether it exists or not, I agree we should aim to do it in a follow-up and include it as part of our presentation.

Reply:
> Do you mean interface parity as stated in the docs, or do you want to describe feature parity in a separate section?

I'm not sure I understand the question here, but I'll try answering. I imagined it like a table inserted right before this text, but if insert (table or list) is too long, it makes sense to put it in a separate file and link it from here.

> I like the first option, but identifying AMCL features is not that easy in my opinion

It doesn't need to be perfect. I think a table-shaped version of https://github.com/Ekumen-OS/beluga/issues/85 will do just fine.

Reply:
Understood, I prefer to work on that in a follow-up PR, I have few cycles to spend on this today.

Reply:
SGTM. This can be merged as is, AFAIC.